services/pipenv-helpers.js

```diff
@@ -0,0 +1,49 @@
+'use strict'
+
+const Joi = require('@hapi/joi')
+const { InvalidParameter } = require('.')
+
+const isDependency = Joi.object({
+  version: Joi.string().required(),
+}).required()
+
+const isLockfile = Joi.object({
+  _meta: Joi.object({
+    requires: Joi.object({
+      python_version: Joi.string().required(),
+    }).required(),
+  }).required(),
+  default: Joi.object().pattern(Joi.string().required(), isDependency),
+  develop: Joi.object().pattern(Joi.string().required(), isDependency),
+}).required()
+
+function getDependencyVersion({
+  kind = 'prod',
+  wantedDependency,
+  lockfileData,
+}) {
+  let dependenciesOfKind
+  if (kind === 'dev') {
+    dependenciesOfKind = lockfileData.develop
+  } else if (kind === 'prod') {
```

Comment:
The pipenv lockfile and docs refer to 'default' and 'dev' dependencies. As opposed to 'prod' and 'dev'. Lets use that terminology here too.

Reply:
Are you thinking in the badge URLs or just internally?

I don't mind using `default` though the pipenv lockfile uses `develop`. I'd prefer to use `dev` in the URL for more cross-project consistency, though I don't feel that strongly about it.

Another option is to translate from the URL terms to the Pipenv terms in `handle()`… which isn't a bad idea, though if we do end up using this function for another service (e.g. "Pipfile.lock from URL") that code would end up being duplicated so we'd probably need a second helper function.

Reply:
I was just thinking about internally using 'default' here instead of 'prod'.

I agree that `:user/:repo/:packageName` should be the user-facing URL. You shouldn't have to specify `:user/:repo/default/:packageName`. Agree, `dev` is fine in the user-facing URL, they are `[dev-packages]` in the Pipfile, the flag is `--dev`, etc https://pipenv.readthedocs.io/en/latest/cli/#cmdoption-pipenv-install-d